examples/colour-uniform/main.rs

```diff
@@ -105,13 +105,13 @@ const COLOR_RANGE: i::SubresourceRange = i::SubresourceRange {
 
 trait SurfaceTrait {
     #[cfg(feature = "gl")]
-    fn get_window_t(&self) -> &back::glutin::WindowedContext;
+    fn get_window_t(&self) -> &back::glutin::WindowedContext<back::glutin::PossiblyCurrent>;
```

Comment:
It feels like someone has toyed with type classes a bit too much... (not you, I'm talking about glutin authors)

Reply:
Yeah, seconded. 😅